client/landing/gutenboarding/hooks/use-step-navigation.ts

```diff
@@ -45,7 +46,11 @@ export default function useStepNavigation(): { goBack: () => void; goNext: () =>
 	const { onSignupDialogOpen } = useSignup();
 	const handleSiteCreation = () =>
 		currentUser
-			? createSite( currentUser.username, freeDomainSuggestion, undefined, shouldSiteBePublic )
+			? createSite( currentUser.username, freeDomainSuggestion, undefined, shouldSiteBePublic, {
```

Comment:
should we set this option in `createSite` or expose a higher-level function? It seems like there are a lot of places we call `createSite` which each look exactly the same. (i.e. it seems like it would be hard to keep them all in sync.)

Reply:
I was thinking the same, it might be good to consolidate this logic that we are reusing in several areas to be a bit more maintainable in the future.

Reply:
Yeah, it's tricky. There has been some consolidation of call sites of `createSite` (e.g. into the `useStepNavigation` hook), but there's still 3 of them left. I think my preferred solution would be to consolidate them even more :thinking: